bitcanna/deploy.yml

```diff
@@ -39,11 +39,6 @@ profiles:
           #     persistent: true
   placement:
     dcloud:
-      attributes:
-        host: akash
-      signedBy:
-        anyOf:
-          - akash1365yvmc4s7awdyj3n2sav7xfx76adc6dnmlx63
```

Comment:
please keep these.
each project has them set as this helps to default the deployments to tested/curated providers.